### ISSUE TITLE
[QOL] Renaming World Saves, Deleting Worlds and Blueprints

### ISSUE THREAD

**Issue by ProPeach** [tsunamayo/Starship-EVO#5324]
More of a QOL addition than anything for sure, but important none the less I think.

Being able to rename and delete world saves from in-game, as well as deleting blueprints in-game too. At the moment to do either you have to know where the files are stored in hidden folders and change those names which isn't really industry standard.

**Comment by tsunamayo** [comment]
@ProPeach I do not wish to bring back the Quit to Title, because I think the number 1 reason people are quitting is because they got a bug. If you get a bug you should always quit to desktop, I think it make newcomers think that the game is a bug mess of epic proportion instead of the truth, which is the game is a bug mess of great proportion.

**Comment by ZachZent** [comment]
> 
@ProPeach I do not wish to bring back the Quit to Title, because I think the number 1 reason people are quitting is because they got a bug. If you get a bug you should always quit to desktop, I think it make newcomers think that the game is a bug mess of epic proportion instead of the truth, which is the game is a bug mess of great proportion.


I'm sorry, but that's awful logic to not include something so standard to basically every video game ever. You will get farrrrrr more complaints about people who wants to switch to different multiplayer servers (in the future) or load up a different world with a planet they like or certain build they are working on than those who will think the game is bugged because they quit to title. They will think it's bugged because it's bugged.

Just do what every early-access/pre-alpha/etc. game does and add a warning message to quit to desktop if they experience a bug with a link to the Github. There is no need to inconvenience players to turning on and off the game. Plus it will get more people to post actual bug reports

**Comment by ProPeach** [comment]
@tsunamayo I agree with Zach completely, this is not a productive way to handle the situation. It could be that from your perspective as the one people go to with a bug that the Quit to Title button is mainly used for that, but that ignores the myriad of regular use cases.

I was also going to suggest the warning. Just a simple popup/rollover on the Quit to Title button saying something like 
"Please do not Quit to Title if you are experiencing a bug! Be sure to Quit to Desktop instead and report the bug _**here**_" (with embedded Github link)

If you simply remove the button, players will come away with the feeling that the game is simply missing a widespread feature and focus on that - they will have no idea you're using it as a strategy to reduce their experience of bugs. Educating the player on _why_ things are the way they are will likely earn the game more respect and empathy than if it simply presents as limited and clunky. Plus it'd be another reminder of the fact that Github exists which would be great as I feel bugs so often go unreported. Subnautica for example had a link to their bug reporting page clearly advertised in the pause menu when it was in EA, it should be presented to players as often as possible.

_________________________________________________________________________________________________________________________________________

On the topic of bugs, how do they factor into your race for Survival Mode? From the looks of it the plan is to get a decent Survival experience before a bigger advertising push to new players, so that new experience will be critical to player retention and growth. Are you planning a big bug fixing/QOL push before this point? All this hard work put into Survival features would really benefit from cleaning up bugs in other areas which could leave a bad impression otherwise.

May the Bug Hunt continue 🐛🔨
